<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importmachine</class>
 <widget class="QDialog" name="importmachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Machine</string>
  </property>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="file_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32765</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>file_line</tabstop>
  <tabstop>browse_button</tabstop>
  <tabstop>confirm_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
